jest-preset.js

```diff
@@ -21,7 +21,14 @@ export default {
       },
     ],
   },
-  collectCoverageFrom: ['<rootDir>/src/**', '!<rootDir>/src/**/*.spec.ts', '!<rootDir>/src/**/*.test.ts'],
+  collectCoverageFrom: [
+    '<rootDir>/src/**',
+    '!<rootDir>/src/**/*.spec.ts',
+    '!<rootDir>/src/**/*.test.ts',
+    '!<rootDir>/src/**/swagger.ts',
+    '!<rootDir>/src/**/swagger.validator.ts',
+    '!<rootDir>/src/**/swagger.guard.ts',
```

Comment:
I'm still uncomfortable with this change.  it's oddly specific to openapi for a general-purpose jest-config imo.  need to marinate on it more.

Reply:
that makes sense

Reply:
even `swagger.ts` is managed in each repo's `.gitignore`
totally agree that it's not ideal to push special stuff into general purpose library.
cloing this PR now.

btw I was confused by the publish-beta.yml, just realized that it's using a bit different approach comparing to existing services :)

Reply:
yeah this uses the new tag/release publish workflow :)